src/validation.cpp

```diff
@@ -961,18 +961,15 @@ bool MemPoolAccept::PreChecks(ATMPArgs& args, Workspace& ws)
 
     ws.m_iters_conflicting = m_pool.GetIterSet(ws.m_conflicts);
 
-    ws.m_ancestors = m_subpackage.m_changeset->CalculateMemPoolAncestors(ws.m_tx_handle);
+    ws.m_parents = m_pool.GetParents(*ws.m_tx_handle);
```

Comment:
In commit "Rework RBF and TRUC validation"

The `GetParents` function is only introduced in the next commit.